packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFGeneratedWebKitApis.m

```diff
@@ -1673,46 +1707,10 @@ @implementation FWFNSObjectFlutterApiCodecReader
 - (nullable id)readValueOfType:(UInt8)type {
   switch (type) {
     case 128:
-      return [FWFNSErrorData fromList:[self readValue]];
```

Comment:
These are removed because the class method `observeValue` now returns an `ObjectOrIdentifier`. When it returned an `Object`, all of these were included as possible return values.